### ISSUE TITLE
Documentation needed

### ISSUE THREAD

**Issue by jacione** [jacione/lauepy#8]
At some point, I need to go through and add docstrings to all of the major functions and files. That would help with collaboration.

**Comment by cheddacheasy** [comment]
For the documentation you desire would you also like function annotation and type hinting?

**Comment by jacione** [comment]
There's still a lot of documentation missing from the source code itself, but I just added a pretty hefty README that should at least help people use the software.

**Comment by cheddacheasy** [comment]
Would you want some help in creating documentation in the source code?

**Comment by jacione** [comment]
Yes, but not yet. We're currently preparing a publication on this technique for x-ray diffraction analysis. Until that's published, we'd rather keep all contributions within the research group. You're welcome to start if you want (I don't think I'll get around to it so it really would be appreciated), but it may be a couple of months before it gets merged.
